conf.json

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
ok, no clue what was it and why now it's gone :D but I would like to know what was doing this guy 🤔

Reply:
That was the configuration for jsdoc, which I never used.
`npm run docs` would have used this config.